rosco-core/src/test/groovy/com/netflix/spinnaker/rosco/providers/aws/AWSBakeHandlerSpec.groovy

```diff
@@ -354,6 +358,71 @@ class AWSBakeHandlerSpec extends Specification {
       1 * packerCommandFactoryMock.buildPackerCommand("", parameterMap, "$configDir/$awsBakeryDefaults.templateFile")
   }
 
+  void 'produces packer command with all required parameters for amzn, with sudo using runAsRoot:true'() {
+    setup:
```

Comment:
It might not be possible to do, but would be nice if the variations of producing packer commands could be factored out into a common function that has the boiler plate and templates so each specialized test just gives it how it is different than the others. That would clarify what the differences are and probably be more maintainable.

I've suggested this type of thing before and sometimes it turns out not to be possible. Dont kill yourself trying to do it, but see if it is reasonable to do.